lib/kuzzle/kuzzle.js

```diff
@@ -48,23 +48,38 @@ const SecurityModule = require('../core/security');
 const RealtimeModule = require('../core/realtime');
 const InternalIndexHandler = require('./internalIndexHandler');
 
+let _kuzzle = null;
+
+Reflect.defineProperty(global, 'kuzzle', {
+  configurable: true,
+  enumerable: false,
+  get () {
+    if (_kuzzle === null) {
+      throw new Error('Kuzzle instance not found. Did you try to use a live-only feature BEFORE starting your application?');
```

Comment:
The capital `BEFORE` seems a little bit harsh to me :sweat_smile: 
```suggestion
      throw new Error('Kuzzle instance not found. Did you try to use a live-only feature before starting your application?');
```